pro/cypress/e2e/migrations/desk.cy.ts

```diff
@@ -0,0 +1,190 @@
+describe('Desk (Guichet) feature', () => {
+  let login: string
+  const password = 'user@AZERTY123'
+
+  beforeEach(() => {
+    cy.visit('/connexion')
+    cy.request({
+      method: 'GET',
+      url: 'http://localhost:5001/sandboxes/pro/create_pro_user_with_bookings',
+    }).then((response) => {
+      login = response.body.user.email
+    })
+  })
+
+  it('I should see help information on desk page', () => {
+    cy.stepLog({ message: 'I am logged in with account' })
+    cy.login({
+      email: login,
+      password: password,
+      redirectUrl: '/',
+    })
+
+    cy.stepLog({ message: 'I go to the "Guichet" page' })
+    cy.url().then((urlSource) => {
+      cy.findAllByText('Guichet').first().click()
+      cy.url().should('not.equal', urlSource)
+    })
```

Comment:
C'est la première fois qu'on teste l'url comme ça. Pourquoi ?
Il y a une raison de tester que la page a bien changé ?

Reply:
c'était une étape un peu générique, mais on va plutôt vérifier qu'on atteint exactement la bonne page plutôt que vérifier qu'on est plus sur la même.
Je corrige ça

Reply:
corrigé. Mais ça a toujours été comme ça, et c'est encore dans `step-definitions/commonSteps.cy.ts` voir `Given('I go to the {string} page')` qui sera supprimé quand la migration finie